test/Datadog.Trace.TestHelpers/MockTracerAgent.cs

```diff
@@ -85,13 +85,11 @@ public MockTracerAgent(int port = 8126, int retries = 5)
             int count,
             int timeoutInMilliseconds = 20000,
             string operationName = null,
-            DateTime? minDateTime = null,
+            DateTimeOffset? minDateTime = null,
```

Comment:
This fixes integration tests failing in Paris because `DateTime.MinValue` is treated as being local, which is a negative value in UTC.